packages/core/src/PlayerContextProvider.js

```diff
@@ -331,7 +338,7 @@ export class PlayerContextProvider extends Component {
     // if not, then load the first track in the new playlist, and pause.
     return {
       ...baseNewState,
-      ...getGoToTrackState(prevState, 0, false)
+      ...getGoToTrackState(prevState, 0, false, true)
```

Comment:
I see why you're doing this, but I'm debating in my head whether it's necessary since this code should only ever be run if it *won't* be the same track, right? I don't have a super strong opinion on this, so I'll trust your judgment.

Reply:
It isn't necessary, as it would reload due to `prevSources[0].src !== newSources[0].src` condition.